### PR TITLE
Add qwen model v2.5

### PR DESCRIPTION
**Pull request by dsudhakarTT**

### CODE REVIEW

File: forge/test/models/pytorch/text/qwen/test_qwen_version_two.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "Qwen/Qwen2.5-14B",
+    "Qwen/Qwen2.5-14B-Instruct",
+    "Qwen/Qwen2.5-32B",
+    "Qwen/Qwen2.5-32B-Instruct",
+    "Qwen/Qwen2.5-72B",
+    "Qwen/Qwen2.5-72B-Instruct",
+]
+
+
+@pytest.mark.parametrize("variant", variants, ids=variants)
```
Comment (dgolubovicTT) [inline forge/test/models/pytorch/text/qwen/test_qwen_version_two.py:28]: Do all variants pass compile? If no, then we should have xfail as parameter of the test...

Comment (vkovinicTT) [reply forge/test/models/pytorch/text/qwen/test_qwen_version_two.py:28]: Do we really need to test all the variants, I mean there are models with 72B params!

Comment (vkovinicTT) [inline forge/test/models/pytorch/text/qwen/test_qwen_version_two.py:28]: Just put `@pytest.mark.nightly` :))

Comment (vkovinicTT) [reply forge/test/models/pytorch/text/qwen/test_qwen_version_two.py:28]: Do we really need all variants (there are 72B models!!)?

Comment (dsudhakarTT) [reply forge/test/models/pytorch/text/qwen/test_qwen_version_two.py:22]: Yeah, I've removed some of the larger variants

Comment (dsudhakarTT) [reply forge/test/models/pytorch/text/qwen/test_qwen_version_two.py:22]: Added :))

**Review by dgolubovicTT — APPROVED** [review]

**Comment by nvukobratTT** [comment]
@dsudhakarTT is this PR still relevant?

**Comment by dsudhakarTT** [comment]
> @dsudhakarTT is this PR still relevant?

Yeah, I'll check the failures and push it again

**Pull request merged**